solutions/instances/provider.tf

```diff
@@ -4,7 +4,7 @@
 
 provider "ibm" {
   ibmcloud_api_key = var.ibmcloud_api_key
-  region           = local.scc_instance_region
+  region           = var.scc_region
```

Comment:
I think this should still be `local.scc_instance_region` - why would we change it?